essential-REST.md

```diff
@@ -0,0 +1,56 @@
+After listening to shows #185 and #189, both on APIs, I started realizing that perhaps there still isn't one central resource for learning about REST APIs. So, I've put together this essential reading list for those of you looking for some of the best resources out there on RESTful APIs.
+
+**[Objects at REST, Part 1: What Is It?](https://medium.com/@Stuart_Weir/objects-at-rest-part-1-what-is-it-3e26f0978616#.rr8x0ahz0)**
+
+This was written recently by me, describing (in one post) many of the fundamental principles, terminology, and ideas behind RESTful APIs. While it's not too granular, I goes into some detail about what REST APIs really are to build your understanding of the technology quickly.
+> An API is an Application Programming Interface. This sounds super technical, but it actually really isn’t. It’s essentially an interface for programmers, much like a User Interface (UI) is an interface for product users.
+
+**[REST API Tutorial](http://www.restapitutorial.com/)**
+
+An excellent resource on REST APIs, REST API Tutorial offers a number of tutorials on REST, quick tips, HTTP method use,resource naming, and a number of other REST API best practices. A _must read_.
+ 
+> Building RESTful web services, like other programming skills is part art, part science. As the Internet industry progresses, creating a REST API becomes more concrete with emerging best practices. As RESTful web services don't follow a prescribed standard except for HTTP, it's important to build your RESTful API in accordance with industry best practices to ease development and increase client adoption.
+
+> Presently, there aren't a lot of REST API guides to help the lonely developer. RestApiTutorial.com is dedicated to tracking REST API best practices and making resources available to enable quick reference and self education for the development crafts-person. We'll discuss both the art and science of creating REST Web services.
+
+> —Todd Fredrich, REST API Expert
+
+**[API Glossary](http://apiglossary.com/), by Mashape**
+
+The Changelog talked a lot about APIs with Ahmad Nassri from Mashape, and after taking a look at some of their free resources, I'm pretty convinced they're the King Kong of APIs. API Glossary is exactly what it sounds like: an open source API Glossary, contributed by Mashape and the larger open source community.
+> Learn the lingo of Application Programming Interfaces and the acronyms, design patterns and tools that surround them.
```

Comment:
It might be more useful here to pull out one of their definitions as an example of the content.